extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -153,7 +156,7 @@ export class AmpSidebar extends AMP.BaseElement {
       this.fixIosElasticScrollLeak_();
     }
 
-    if (this.isOpen_()) {
+    if (element.hasAttribute('open')) {
```

Comment:
Do you also need to set `this.opened_ = true` either here or in `open_()`? I don't see the change where the initial state from the `open` attribute sets the `this.opened_` variable to true.

Reply:
As discussed off-line, this doesn't seem to have worked for a long time and never validated.